approve.py

```diff
@@ -9,12 +9,12 @@
 import yaml
 from celery import Celery
 from Crypto.PublicKey import RSA
+from ocflib.account import CREATE_PUBLIC_KEY
```

Comment:
This needs to be `from ocflib.account.creation import CREATE_PUBLIC_KEY`, but otherwise looks good!